fdbrpc/genericactors.actor.h

```diff
@@ -78,11 +78,7 @@ Future<Void> tryInitializeRequestStream(RequestStream<Req>* stream, Hostname hos
 	if (!address.present()) {
 		return Void();
 	}
-	if (stream == nullptr) {
-		stream = new RequestStream<Req>(Endpoint::wellKnown({ address.get() }, token));
-	} else {
-		*stream = RequestStream<Req>(Endpoint::wellKnown({ address.get() }, token));
-	}
+	*stream = RequestStream<Req>(Endpoint::wellKnown({ address.get() }, token));
```

Comment:
Consider adding a nullptr check

Reply:
This pointer will never be nullptr. The ideal code style here is passing by reference, but it doesn't work with ACTOR. I checked error_definitions.h but didn't find any error can describe this case accurately.

Reply:
Can you add `ASSERT(stream != nullptr)` at the beginning?

nullptr will cause the program to stop.

This helps future debugging if `tryInitializeRequestStream` is used by others in a wrong way. Otherwise, this may potentially be a trap for a segfault that is harder to find.

Reply:
Good point.